src/main/java/gameframework/drawing/GameUniverseViewPortDefaultImpl.java

```diff
@@ -10,11 +10,16 @@
 import java.util.Iterator;
 
 public class GameUniverseViewPortDefaultImpl implements GameUniverseViewPort {
+
 	private Image buffer;
 	protected BackgroundImage background;
 	protected GameData data;
 
-	protected URL backgroundImage() {
+	protected URL backgroundImage(String path) {
+		return this.getClass().getResource(path);
+	}
+
+	protected URL backgroundImage() { // with default pathname
```

Comment:
please remove this Java comment as it is obvious. Better, please replace it with proper Javadoc.